.vitepress/config.mts

```diff
@@ -150,6 +150,14 @@ export default defineConfig({
                   text: 'Logging a user out of the system',
                   link: '/api-specification/auth-controller/logging-a-user-out-of-the-system.md',
                 },
+                {
+                  text: 'Toggling 2FA',
```

Comment:
I remember we decided those were not needed, a client would use the web app for 2fa anyways